lib/install/validate/validator.go

```diff
@@ -127,20 +118,60 @@ func CreateNoDCCheck(ctx context.Context, input *data.Data) (*Validator, error)
 
 	v.Session.Populate(ctx)
 
+	// only allow the datacenter to be specified in the taget url, if any
```

Comment:
session.populate will search datacenter, if don't set before populate, there will have datacenter not set error.